systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -886,7 +886,7 @@ def forward_vr(self, rule):
         self.fw.append(["filter", "", fw7])
 
     def forward_vpc(self, rule):
-        fw_prerout_rule = "-A PREROUTING -d %s/32 -i %s" % (rule["public_ip"], self.getDeviceByIp(rule['public_ip']))
+        fw_prerout_rule = "-A PREROUTING -d %s/32 " % (rule["public_ip"])
```

Comment:
@rhtyd This is already fixed in `forward_vr` as there is only one internal network for each VR so the IPtables rules work correctly there.